_projects/04-sip-process-reloaded.md

```diff
@@ -1,10 +1,10 @@
 ---
 label: sip
-name: Scala improvement process lead
+name: Scala improvement process
```

Comment:
This change is so that we can reuse the projects for mere contributors like me 😄